utils/common.js

```diff
@@ -81,8 +81,27 @@ function looksLikeId(key) {
   return constants.POSTGRES_UUID_RE.test(key);
 }
 
+/**
+ * Validates that atleast one of the given fields is present in object.
+ * @param  {Object} reqBody - Request body
+ * @param  {Array} fieldsArr - Fields array
+ * @throws {ValidationError} - If none of the given fields are presnt in object
+ */
+function validateAtleastOneFieldPresent(reqBody, fieldsArr) {
```

Comment:
I think I'd like to see some unit tests for this function in isolation, testing all the combinations of reqBody with no keys/some keys/all keys, and fieldArr with no elements, one element, multiple elements, and null or undefined for each arg, etc.

Comprehensive unit testing of the helper function means we don't have to test all the different permutations every time some other module uses this fn.